lib/matplotlib/backend_bases.py

```diff
@@ -1121,7 +1121,7 @@ def get_antialiaseds(self):
         return self._antialiaseds
 
     def get_capstyles(self):
-        return [cs.name for cs in self._capstyles]
+        return self._capstyles
```

Comment:
Revert this change. I think it should be the previous statement.
```suggestion
        return [cs.name for cs in self._capstyles]
```